android/java/org/mlperf/inference/activities/CalculatingActivity.java

```diff
@@ -1294,6 +1347,7 @@ protected boolean extractFileFail(String src) {
           if (!unZip(tmpFile, dest)) {
             return true;
```

Comment:
Why not delete tmpFile before return if the unzip failed?

Reply:
Fixed